CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -27,10 +27,13 @@
 #include <CoreFoundation/ForFoundationOnly.h>
 #include <fts.h>
 #include <pthread.h>
+
+#if !TARGET_OS_ANDROID
 #include <execinfo.h>
+#endif
 
-#if __has_include(<malloc/malloc.h>)
-#include <malloc/malloc.h>
+#if __has_include(<execinfo.h>)
```

Comment:
What is going on here now? Why is execinfo.h being conditionally included twice?